MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -1367,7 +1367,7 @@ private void PatchGrabbable(bool? grabbable)
 						}
 					}
 				}
-
+				((ITargetBehavior)behaviorComponent.Behavior).Grabbable = grabbable.Value;
```

Comment:
Can we assume this is an ITargetBehavior? This actor could have a tool behavior on it, right?

Reply:
I think for the case when there is no rigid body it should be safe to assume (since this was also the case before physics bridge integration). Maybe then this should be then in an else branch with some additional checks (?)

Reply:
Now I separated the rigid-body case and the non-rigid body case as it was before physics bridge.